snuba/datasets/outcomes.py

```diff
@@ -8,4 +9,7 @@ class OutcomesDataset(Dataset):
     """
 
     def __init__(self) -> None:
-        super().__init__(default_entity=EntityKey.OUTCOMES)
+        super().__init__(
+            default_entity=EntityKey.OUTCOMES,
+            quota_control=ProjectQuotaControl("project_id"),
```

Comment:
Outcomes only require an org_id, so I don't know if they even pass the project_id as a parameter.

Reply:
Yes I am aware. THe project rate limiter is already broken there, so the idea would be to separately fix that